src/inc/schedule-output-functions.php

```diff
@@ -274,15 +326,24 @@ function wpcs_schedule( $atts, $content ) {
  * @return array
  */
 function wpad_draw_session( $talk, $is_current, $text, $session_id ) {
-	$talk_ID   = $talk['id'];
-	$datatime  = $talk['ts'];
-	$mins      = gmdate( 'i', get_post_meta( $talk_ID, '_wpcs_session_time', true ) );
-	$time      = gmdate( 'H', get_post_meta( $talk_ID, '_wpcs_session_time', true ) );
-	$time_html = '<div class="talk-header"><h2 class="talk-time" data-time="' . $datatime . '" id="talk-time-' . $time . '"><div class="time-wrapper"><span>' . $time . ':' . $mins . ' UTC<span class="screen-reader-text">,&nbsp;</span></span>' . ' </div></h2><div class="talk-wrapper">%s[control]</div></div>';
-	$talk_type = sanitize_html_class( get_post_meta( $talk_ID, '_wpcs_session_type', true ) );
-	$speakers  = wpcs_session_speakers( $talk_ID, $talk_type );
-	$sponsors  = wpcs_session_sponsors( $talk_ID );
-	$talk      = get_post( $talk_ID );
+	$talk_ID         = $talk['id'];
+	$datatime        = $talk['ts'];
+	$mins            = gmdate( 'i', get_post_meta( $talk_ID, '_wpcs_session_time', true ) );
+	$time            = gmdate( 'H', get_post_meta( $talk_ID, '_wpcs_session_time', true ) );
+	$tag_heading_txt = __( 'Tags', 'wpad' );
+	$tags_html       = wpad_get_tags_html( $talk_ID );
+	$track_name      = wpad_get_track_name( $talk_ID );
+	$track_name_html = '';
+
+	if ( ! empty( $track_name ) ) {
+		$track_name_html = '<h2 class="talk-track">' . $track_name . '</h2>';
```

Comment:
This should not be an `h2` heading.  Let's move the text so it's after the talk title, and link it to the archive page for the track, with no heading. Give it a prefix like "Level:". 

The archive page probably doesn't have any design; we haven't used it previously, but that's something we can work on separately.